client/src/utils.ts

```diff
@@ -0,0 +1,27 @@
+import * as vscode from 'vscode';
+
+export const isWindows = process.platform === 'win32';
+
+function isPowershell() {
+  try {
+    const config: vscode.WorkspaceConfiguration = vscode.workspace.getConfiguration(
+      'ElmLS',
+    );
+    const t: string = <string>config.get('terminal.integrated.shell.windows');
```

Comment:
Is this provided by vscode?

Reply:
Yeah, `terminal.integrated.shell.windows` is one of the built-in Code settings